rust/main/agents/relayer/src/msg/pending_message.rs

```diff
@@ -716,7 +719,7 @@ impl PendingMessage {
                         "Message has been retried too many times, skipping",
                     );
                 }
-                u64::MAX
+                chrono::Duration::weeks(10).num_seconds() as u64
```

Comment:
more for my learning, why chrono instead of std time?

Reply:
`std::time` doesn't have a `weeks` method (at least on the version we're on)